packages/apollo-server-integration-testsuite/src/index.ts

```diff
@@ -1070,7 +1070,7 @@ export default (createApp: CreateAppFunc, destroyApp?: DestroyAppFunc) => {
         it('calls serverWillStart before serving a request', async () => {
```

Comment:
@abernix Thoughts on how to make this test work?

Reply:
Trying out a new delaying approach, lmk what you think. Not sure if its awful or amazing.